src/components/ui/Button.tsx

```diff
@@ -227,7 +227,7 @@ export default function ({
           {gradientFont ? (
             <span className={textGradient(available)}>{children}</span>
           ) : (
-            <div>{children}</div>
+            <>{children}</>
```

Comment:
That's really strange, we have `flex flex-row gap-x-2` in the styles, but `gap` will never work because you wrap this into div

I double checked, nothing breaks